docs/lib/debugging/fragments/native_common/dev-menu/common.md

```diff
@@ -0,0 +1,38 @@
+When running a debug build of your app, you can access information, logs, 
+and file issues to Amplify GitHub Issues directly from your application through the developer menu. 
+By filing GitHub issues through the developer menu, critical information surrounding the issue is 
+automatically added to the issue description, allowing Amplify community members to better assist 
+you in resolving the issue.
+
+<docs-filter platform="android">
+
+![Amplify Android Developer Menu](~/images/debugging/androidDevMenu.png)
+
+</docs-filter>
+
+<amplify-callout warning>
+
+The developer menu is disabled in production builds.
```

Comment:
maybe worth bolding "production"

Reply:
Be more explicit about "production app" builds. Could be confusing in some cases if it's the "production" Amplify build or "production" customer app build.